atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/SweepTaskRunner.java

```diff
@@ -176,200 +153,67 @@ private SweepResults doRun(TableReference tableRef,
         //     we encounter
         long sweepTs = sweeper.getSweepTimestampSupplier().getSweepTimestamp(
                 unreadableTimestampSupplier, immutableTimestampSupplier);
-
-        byte[] startRow = MoreObjects.firstNonNull(nullableStartRow, PtBytes.EMPTY_BYTE_ARRAY);
-        RangeRequest range = RangeRequest.builder()
+        CandidateCellForSweepingRequest request = ImmutableCandidateCellForSweepingRequest.builder()
```

Comment:
is there a reason the SweepingRequest includes the row and not the column? if a batch stops in the middle of the row does that mean we continue to reexamine from the start of the row again?